### PR TITLE
GHA/windows: use libuv for event-based tests on openssl job

### PR DESCRIPTION
**Pull request by talregev**
Use libuv for event-based tests on openssl job.

### CODE REVIEW

**Comment by vszakats** [comment]
Would it make sensel to also enable event-based tests for this job? It can be enabled by adding `-e` to tflags.

**Comment by vszakats** [comment]
Thanks for making a try with the tests, .. but seeing the slow performance on Windows, I think it's best to keep it disabled there.

**Comment by talregev** [comment]
> Thanks for making a try with the tests, .. but seeing the slow performance on Windows, I think it's best to keep it disabled there.

ok.

**Review by vszakats — APPROVED** [review]

File: .github/workflows/windows.yml [hunk 1]
```diff
@@ -545,7 +545,7 @@ jobs:
 
       - name: 'cmake run tests'
         if: ${{ matrix.tflags != 'skipall' && matrix.tflags != 'skiprun' }}
-        timeout-minutes: 16
+        timeout-minutes: 60
```
[the remark this answers is not in the document]
Comment (talregev) [reply .github/workflows/windows.yml]: ok.

**Pull request closed without merging**